hathor/consensus/block_consensus.py

```diff
@@ -161,16 +161,9 @@ def update_voided_info(self, block: Block) -> None:
             self.mark_as_voided(block, skip_remove_first_block_markers=True)
 
             # Get the score of the best chains.
-            heads = [cast(Block, storage.get_transaction(h)) for h in storage.get_best_block_tips()]
-            best_score: int | None = None
-            for head in heads:
-                head_meta = head.get_metadata(force_reload=True)
-                if best_score is None:
-                    best_score = head_meta.score
-                else:
-                    # All heads must have the same score.
-                    assert best_score == head_meta.score
-            assert best_score is not None
+            head = storage.get_best_block()
```

Comment:
Does it assert that there's only one head?

Reply:
It doesn't, and it can't unless it uses the block-tips indexes that will be removed.

Reply:
I think this thread can be closed.